wasmaudioworklet/audioworkletpolyfill.js

```diff
@@ -6,12 +6,21 @@
  */
 
 if(typeof AudioContext !== 'function') {
-    window.AudioContext = window.webkitAudioContext;
+    /** @type {any} */
+    const win = window
+    window.AudioContext = win.webkitAudioContext;
 }
 
 if(typeof AudioWorkletNode !== 'function') {
     console.log('No audioworklet support - using polyfill');
-    window.AudioWorkletNode = function(context, processorName) {
+
+    window.AudioWorkletNode = /** @type {AudioWorkletNode} */(AudioWorkletNodePolyfill)
```

Comment:
This syntax is a type cast. The parentheses are required here. `/** @type {any} */(AudioWorkletNodePolyfill)` would be shorter and would also work (we don't care much about types here anyways).